feature/settings/push/src/main/kotlin/app/k9mail/feature/settings/push/ui/PushFoldersActivity.kt

```diff
@@ -0,0 +1,62 @@
+package app.k9mail.feature.settings.push.ui
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.activity.result.contract.ActivityResultContract
+import app.k9mail.core.ui.compose.theme.K9Theme
+import com.fsck.k9.Account.FolderMode
+import com.fsck.k9.ui.base.K9Activity
+
+/**
+ * Screen to change the "Push folders" setting of an account.
+ */
+class PushFoldersActivity : K9Activity() {
```

Comment:
As I suspect this is a temporary solution, the new Activity could be the simpler solution. For future changes it would be good to start limiting the use of activities and use the `FeatureLauncherActivity` and it's navigation graph instead. This helps to prepare the project to migrate to Jetpack Compose and avoid behaviour that needs to be rewritten later again.